com.chartboost.mediation/Editor/ChartboostMediationSettingsEditor.cs

```diff
@@ -48,7 +48,7 @@ private void SetupUI()
 			EditorGUILayout.BeginVertical();
 			EditorGUILayout.LabelField(_partnerKilLSwitchTitle, _title);
 			EditorGUILayout.Space();
-			EditorGUILayout.HelpBox("Select partners to disable their initialization.", MessageType.Info);
+			EditorGUILayout.HelpBox("Select partners to disable their initialization.\n(Enum flag has been deprecated and will be removed in future versions, use StartWithOptions instead)", MessageType.Info);
```

Comment:
Should we remove the `use StartWithOptions instead` part?  Or:

```suggestion
			EditorGUILayout.HelpBox("Select partners to disable their initialization.\n(Enum flag has been deprecated and will be removed in future versions, use the StartWithOptions API instead)", MessageType.Info);
```